src/App.js

```diff
@@ -13,41 +13,46 @@ class App extends Component {
 
   state = {
     loggedIn: JSON.parse(localStorage.getItem('loggedIn') || 'false'),
-    username: localStorage.getItem('username') || ''
+    username: localStorage.getItem('username') || '',
+    id : localStorage.getItem('id') || ''
   }
 
   handleLogin = character => {
     this.setState({
       loggedIn: true,
-      username: character['username']
+      username: character['username'],
+      id: character['id']
     }, () => {
     localStorage.setItem('loggedIn', true)
-    localStorage.setItem('username', character['username'])      
+    localStorage.setItem('username', character['username'])
+    localStorage.setItem('id', character['id'])       
   })
   }
 
   handleLogout = () => {
     this.setState({
       loggedIn: false,
-      username: ''
+      username: '',
+      id: ''
     }, () => {
       localStorage.setItem('loggedIn', false)
       localStorage.setItem('username', '')
+      localStorage.setItem('id', '')
     })
   }
 
   render () {
     return (
       <Router>
         <div>
-          <Nav loggedIn={this.state.loggedIn} loggedUsername={this.state.username} handleLogout={this.handleLogout}/>
+          <Nav loggedIn={this.state.loggedIn} loggedUsername={this.state.username} loggedId={this.state.id} handleLogout={this.handleLogout}/>
```

Comment:
Instead of passing all different attributes of the 'character' (which should probably just be called user), it probably makes more sense to pass the user and then just access the attributes as you need them. I can update this in my change.

Reply:
sounds good!